requirements.txt

```diff
@@ -1,90 +1,85 @@
-Sphinx==1.6.5
+alabaster==0.7.10
```

Comment:
the changes this file breaks the separation that previously existed, can you follow these two steps in order to retain that?

```
# Installing the top-level dependencies, upgrading them if necessary
pip install –r requirements-top-level.txt --upgrade 

# Updating requirements.txt
pip freeze –r requirements-top-level.txt > requirements.txt
```

Reply:
The main issue I foresee with this is that dev requirements are lost with this workflow

Reply:
@ryandeivert how's that? developers just `pip install requirements.txt` as normal

Reply:
ah sorry I didn't realize the dev/test requirements were also in the normal requirements file. we should definitely move away from that

Reply:
@jacknagz, that is exactly what I did, with the exception that I didn't order the requirements with the `-r` flag to `pip freeze`. So it was just the requirements order that changed (the files have the same meaning and contents).

I've put the order back, thanks for listing the command!